jetpackperformancescanner/src/main/java/com/qamar/composescanner/TrackerModifier.kt

```diff
@@ -47,6 +57,23 @@ fun Modifier.trackRecompositions(): Modifier {
                 }
             }
         )
-        .border(2.dp, Color.Red)
+        .border(2.dp, assignedColor)
         .padding(16.dp)
+}
+
+private fun generateRandomColor(): Color {
+    return Color(
+        red = (0..255).random() / 255f,
+        green = (0..255).random() / 255f,
+        blue = (0..255).random() / 255f,
+        alpha = 1f
+    )
+}
+
```

Comment:
what about using this instead of doing those calculations? 

```kotlin
fun getRandomColor(): Color {
    return Color(
        red = Random.nextFloat(),
        green = Random.nextFloat(),
        blue = Random.nextFloat(),
        alpha = 1f
    )
}
```

Reply:
The first implementation generates a random color while the one you suggested returns a random color from a predefined list of theme colors. Is there a reason for that?

Reply:
I  have concerns that generating those random colors will add an overhead for anyone is using it.  But let's proceed we can have benchmark for it later.